src/log.cpp

```diff
@@ -187,13 +190,48 @@ void set_log_to_file()
 	if(is_log_dir_writable_.value_or(false)) {
 		// get the log file stream and assign cerr+cout to it
 		output_file_path_ = filesystem::get_logs_dir()+"/"+unique_log_filename();
-		static std::unique_ptr<std::ostream> logfile { filesystem::ostream_file(output_file_path_) };
-		static std::ostream cerr_stream{std::cerr.rdbuf()};
-		//static std::ostream cout_stream{std::cout.rdbuf()};
-		auto cerr_tee { boost::iostreams::tee(*logfile, cerr_stream) };
-		output_file_.reset(new boost::iostreams::stream<decltype(cerr_tee)>{cerr_tee, 4096, 0});
-		std::cerr.rdbuf(output_file_.get()->rdbuf());
-		std::cout.rdbuf(output_file_.get()->rdbuf());
+
+		// redirect to a file
+		// IMPORTANT: apparently redirecting stderr/stdout will also redirect std::cerr/std::cout, but the reverse is not true
+		//            redirecting std::cerr/std::cout will *not* redirect stderr/stdout
```

Comment:
I guess this is technically true, but… the way it's stated is maybe a tad misleading.

Redirecting std::cerr/std::cout is in the iostream, replacing their stream buffer with something else. Redirecting stderr/stdout is done at a lower level, essentially making both streams be the same stream.

I'm not sure what's a better way to put this…